frontend/projects/upgrade/src/app/features/dashboard/home/components/experiment-list/experiment-list.component.html

```diff
@@ -216,33 +216,33 @@
           </span>
         </th>
         <td class="ft-12-600" mat-cell *matCellDef="let experiment">
-          <mat-chip-list>
+          <mat-chip-listbox>
             <ng-container *ngFor="let context of experiment.context; let contextIndex = index">
-              <mat-chip
+              <mat-chip-option
```

Comment:
it looks like I missed this in my second commit, these should be `mat-chip` and not `mat-chip-option`, would you mind changing that while you're in there?

Reply:
Done!